src/store/NavigationWidgetStore.js

```diff
@@ -0,0 +1,42 @@
+import { observable, action, when } from 'mobx';
+import { getUTCEpoch } from '../utils';
+
+export default class NavigationWidgetStore {
+    @observable stx;
+    @observable enableHome = false;
+
+    get chart() { return this.mainStore.chart; }
+    get stateStore() { return this.mainStore.state; }
+    get stxx() { return this.chart.stxx; }
+
+    constructor(mainStore) {
+        this.mainStore = mainStore;
+        when(() => this.mainStore.chart.context, this.onContextReady);
+    }
+
+    onContextReady = () => {
+        this.stx = this.mainStore.chart.context.stx;
```

Comment:
@shayanbinary what's the difference between  `context.stx` and `chart.stxx`? both of them are the same.